### DISCUSSION TITLE
simplify each, {}

### DISCUSSION THREAD

**Discussion by yuuyins** [nushell/nushell#7341] [Ideas]
each {||} syntax is cumbersome. imo it is unacceptable for something like a shell that instead should be easy, fast to type/think/remember

maybe getting inspiration from functional langs like haskel, nix

 ```
each ,it something $it . something else
```

**Comment by theAkito** [comment]
> each {||} syntax is cumbersome. imo it is unacceptable for something like a shell that instead should be easy, fast to type/think/remember

Seems to me much more intuitive, as it is used all over languages, rather than the following example you had given.

```
each ,it something $it . something else
```

Don't even know what that means or what those punctuation is doing there. This is really alien. 👽
